numpy/linalg/linalg.py

```diff
@@ -2006,7 +2006,7 @@ def _multi_svd_norm(x, row_axis, col_axis, op):
     """
     if row_axis > col_axis:
         row_axis -= 1
-    y = rollaxis(rollaxis(x, col_axis, x.ndim), row_axis, -1)
+    y = moveaxis(x, [row_axis, col_axis], [-2, -1])
```

Comment:
And in fact, after removing it, there's an assertion for nonequality at the caller